app/api/v1/endpoints/memory/endpoints.py

```
@@ -425,6 +491,151 @@ async def list_relationships(
+                    "occurrenceCount": rel.get("occurrence_count"),
+                    "acknowledgedAt": rel.get("acknowledged_at"),
+                    "lastModifiedAt": rel.get("last_modified_at"),
+                    "hasEditedProvenance": has_edited_provenance_by_link.get(
+                        rel_id, False
+                    ),
+                }
+            )
+
+        return GraphDataResponse(nodes=nodes_payload, links=links_payload)
+    except Exception as exc:
+        logger.exception("Error building graph data: %s", exc)
```

Ruff (0.14.14)

[warning] Redundant exception object included in `logging.exception` call

(TRY401)